README.md

```diff
@@ -20,14 +20,16 @@ Homebrewery is open source, so feel free to clone it, tinker with it, or run you
 1. `npm start`
 
 #### standalone PHB stylesheet
-If you just want the stylesheet that is generated to make pages look like they are from the Player's Handbook, you will find it [here](https://github.com/stolksdorf/homebrewery/blob/master/phb.standalone.css)
+If you just want the stylesheet that is generated to make pages look like they are from the Player's Handbook, you will find it [here](https://github.com/stolksdorf/homebrewery/blob/master/phb.standalone.css).
 
 If you are developing locally and would like to generate your own, follow the above steps and then run `npm run phb`.
 
 ### changelog
 
-You can check out the changelog [here](https://github.com/stolksdorf/homebrewery/blob/master/changelog.md)
+You can check out the changelog [here](https://github.com/stolksdorf/homebrewery/blob/master/changelog.md).
 
 ### license
 
-This project is licensed under [MIT](./license)
+This project is licensed under [MIT](./license). Which means you are free to use The Homebrewery is any way that you want, except for claiming that you made it yourself.

```

Comment:
"free to use The Homebrewery is any way" ⇒ "free to use The Homebrewery **in** any way"

Reply:
Thanks for picking that up.

Reply:
Happy to merge once this is fixed up.

Reply:
I got it. Going to merge this up.